lib/lynx/pipe/run.rb

```diff
@@ -4,7 +4,13 @@ module Lynx
   module Pipe
     class Run < Basic
       def perform(command)
-        system(command.to_s)
+        IO.popen(command.to_s) do |io|
```

Comment:
Would be better to implement this as a new pipe. See `::Lynx::Pipe::Get` and `::Lynx::Pipe::Write` for examples. `Write` might duplicate a lot of this functionality, if not, you could add a `::Lynx::Pipe::Popen` or `IO` or something. The logger can then be defined on that class only.